github/issue.go

```diff
@@ -111,15 +111,31 @@ func (g GitHub) maybeOpinion(issueHook *octokat.IssueHook) error {
 	return nil
 }
 
+func labelFromVersion(version, suffix string) string {
+	switch suffix {
+	// Dev suffix is associated with a master build.
+	case "dev":
+		return "version/master"
```

Comment:
Hence me thinking to possibly add both (version + "master" / "dev" label). Perhaps I'm thinking too much, so I'm ok with merging as-is